### PR TITLE
Document support for agent level tags

### PR DESCRIPTION
**Pull request by annanay25**
Signed-off-by: Annanay <annanay.a@media.net>

## Which problem is this PR solving?
- Documentation for jaegertracing/jaeger#1396

### CODE REVIEW

File: content/docs/1.11/deployment.md [hunk 1]
```diff
@@ -97,6 +97,10 @@ When using gRPC, you have several options for load balancing and name resolution
 * Static list of hostnames and round-robin load balancing. This is what you get with a comma-separated list of addresses. (example: `reporter.grpc.host-port=jaeger-collector1:14250,jaeger-collector2:14250,jaeger-collector3:14250`)
 * Dynamic DNS resolution and round-robin load balancing. To get this behaviour, prefix the address with `dns:///` and gRPC will attempt to resolve the hostname using SRV records (for [external load balancing](https://github.com/grpc/grpc/blob/master/doc/load-balancing.md)), TXT records (for [service configs](https://github.com/grpc/grpc/blob/master/doc/service_config.md)), and A records. Refer to the [gRPC Name Resolution docs](https://github.com/grpc/grpc/blob/master/doc/naming.md) and the [dns_resolver.go implementation](https://github.com/grpc/grpc-go/blob/master/resolver/dns/dns_resolver.go) for more info. (example: `--reporter.grpc.host-port=dns:///jaeger-collector.jaeger-infra.svc:14250`)
 
+### Agent level tags
```
Comment (yurishkuro) [inline content/docs/1.11/deployment.md]: I don't think this feature made it to 1.11

Comment (annanay25) [reply content/docs/1.11/deployment.md]: Ah, my bad

**Comment by yurishkuro** [comment]
Thanks!

**Comment by annanay25** [comment]
😄

**Pull request merged**